src/official-widgets/camera-search/camera-search.spec.tsx

```diff
@@ -901,4 +901,139 @@ describe('camera-search', () => {
 
     expect(counter).toEqual(3);
   });
+
+  it('uploading image should return cropped image', async () => {
+    const widgetClient = getWidgetClient(widgetConfig, 'wigmix_camera_search', 'VERSION', () => ({
+      ...mockVisearchClient,
+      productMultisearch: jest.fn().mockImplementation((_, handler) => {
+        handler(getStandardMultiSearchSuccessWithBoxResponse());
+      }),
+      productMultisearchAutocomplete: jest.fn().mockImplementation((_, handler) => {
+        handler(getStandardMultiSearchAutocompleteResponse());
+      }),
+    }));
+    testComponent = render(
+      <RootContext.Provider value={document.body}>
+        <WidgetDataContext.Provider value={{ widgetConfig, widgetClient, darkMode: false, locale: 'en' }}>
+          <IntlProvider messages={texts['en']} locale='en' defaultLocale='en'>
+            <CameraSearch renderModalWithoutPortal={true} />
+          </IntlProvider>
+        </WidgetDataContext.Provider>
+      </RootContext.Provider>,
+    );
+
+    act(() => {
+      const popupTriggerButton = testComponent.getByTestId('wigmix-popup-trigger-button');
+      popupTriggerButton.click();
+    });
+
+    const makeMockData = (files: File[]): any => ({
+      dataTransfer: {
+        files,
+        items: files.map((file) => ({
+          kind: 'file',
+          type: file.type,
+          getAsFile: () => file,
+        })),
+        types: ['Files'],
+      },
+    });
+
+    const fileInput = testComponent.getByTestId('wigmix-cs-upload-icon-dropzone');
+    const testFile = new File(['image-content'], 'test-file.png', { type: 'image/png' });
+
+    act(() => {
+      fireEvent.drop(fileInput, makeMockData([testFile]));
+    });
+
+    await waitFor(async () => {
+      // Advance time for the FileReader onload function to fire
+      await new Promise((resolve) => {
+        setTimeout(resolve, 500);
+      });
+    }).catch(() => {
+      // Expected to encounter timeout error here; swallow the exception as the test can proceed harmlessly after this
+    });
+
+    act(() => {
+      const productCardImages = testComponent.queryAllByTestId('wigmix-product-card-image');
+      productCardImages.forEach((productCardImage) => {
+        fireEvent.load(productCardImage);
+      });
+    });
+
+    act(() => {
+      const findSimilarButtons = testComponent.queryAllByTestId('wigmix-find-similar-button');
+      findSimilarButtons[1].click();
+    });
+
+    act(() => {
+      const productCardImages = testComponent.queryAllByTestId('wigmix-product-card-image');
+      productCardImages.forEach((productCardImage) => {
+        fireEvent.load(productCardImage);
+      });
+    });
+
+    const inactiveSearchCropImage = testComponent.queryAllByTestId('wigmix-inactive-product-crop');
+    expect(inactiveSearchCropImage.length).toBeGreaterThan(0);
+  });
+
+  it('clicking on gallery image should return cropped image', async () => {
```

Comment:
This is the test of clicking inactive search history, which is already present. Should only need to modify that test so that the initial search returns boxed response, which is tied to a previous comment.